tests/generation/test_utils.py

```diff
@@ -2415,14 +2404,6 @@ def test_generate_inputs_and_encoder_kwargs(self):
         with self.assertRaises(ValueError):
             model.generate(input_ids, input_ids=input_ids)
 
-    def test_generate_too_many_encoder_kwargs(self):
-        article = """I need input_ids to generate"""
-        tokenizer = GPT2Tokenizer.from_pretrained("hf-internal-testing/tiny-random-gpt2")
-        model = GPT2LMHeadModel.from_pretrained("hf-internal-testing/tiny-random-gpt2", max_length=10).to(torch_device)
-        input_ids = tokenizer(article, return_tensors="pt").input_ids.to(torch_device)
-        with self.assertRaises(ValueError):
-            model.generate(input_ids=input_ids, inputs_embeds=input_ids)
```

Comment:
As written in the PR header -- we actually want to pass both, so we also get the prompt in the output (as expected of encoder-decoder models)